pwiz_tools/Skyline/Controls/Graphs/GraphFullScan.cs

```diff
@@ -598,13 +604,125 @@ private void PopulateProperties()
                     var chromSet = stateProvider.DocumentUI.Settings.MeasuredResults.Chromatograms.FirstOrDefault(
                         chrom => chrom.ContainsFile(_msDataFileScanHelper.ScanProvider.DataFilePath));
                     spectrumProperties.ReplicateName = chromSet?.Name;
+                    if (_peaks != null && _peaks.Length > 0)
+                    {
+                        var nodePath = DocNodePath.GetNodePath(_msDataFileScanHelper.CurrentTransition?.Id,

```

Comment:
I think you need to check whether nodePath is null, especially because of that "CurrentTransition?.Id"